services-js/registry-certs/client/store/BirthCertificateRequest.ts

```diff
@@ -0,0 +1,44 @@
+import { observable } from 'mobx';
+
+import { BirthCertificateRequestInformation } from '../types';
+
+// This is used for initial state, and to reset all fields if
+// user selects “start over”
+export const initialRequestInformation: BirthCertificateRequestInformation = {
+  forSelf: null,
+  howRelated: '',
+  bornInBoston: '',
+  parentsLivedInBoston: '',
+  firstName: '',
+  lastName: '',
+  altSpelling: '',
+  birthDate: '',
+  parentsMarried: '',
+  parent1FirstName: '',
+  parent1LastName: '',
+  parent2FirstName: '',
+  parent2LastName: '',
+};
+
+export default class BirthCertificateRequest {
+  @observable quantity: number = 0;
+  @observable
+  requestInformation: BirthCertificateRequestInformation = initialRequestInformation;
+
+  public setQuantity(newQuantity: number): void {
```

Comment:
These setters should be annotated with `@action`. Are you getting warnings in your console?

Reply:
Nope! Is there a mobx debugger extension I should install?

Reply:
I was looking at `DeathCertificatesCart`’s `setQuantity` method when I was working on this, and it didn't use `@action`? Would love to sit down Monday so I can understand this better.

Reply:
Yah let's go through this and see.